store/src/test/java/org/apache/rocketmq/store/schedule/ScheduleMessageServiceTest.java

```diff
@@ -111,14 +111,17 @@ public void init() throws Exception {
 
 
     @Test
-    public void deliverDelayedMessageTimerTaskTest() throws InterruptedException {
+    public void deliverDelayedMessageTimerTaskTest() throws Exception {
         MessageExtBrokerInner msg = buildMessage();
         int realQueueId = msg.getQueueId();
         // set delayLevel,and send delay message
         msg.setDelayTimeLevel(delayLevel);
         PutMessageResult result = messageStore.putMessage(msg);
         assertThat(result.isOk()).isTrue();
 
+        StoreTestUtil.waitCommitLogReput(messageStore);
+        StoreTestUtil.flushConsumeQueue(messageStore);
```

Comment:
StoreTestUtil.flushConsumeQueue(messageStore)  is   not necessary

Reply:
I delete StoreTestUtil.flushConsumeQueue(messageStore);